maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -47,6 +47,8 @@ data class SwipeCommand(
     val startPoint: Point? = null,
     val endPoint: Point? = null,
     val elementSelector: ElementSelector? = null,
+    val startRelative: String? = null,
```

Comment:
This those are strings, you can also update `evaluateScripts` to support JS in there